models/transaction_test.go

```diff
@@ -0,0 +1,41 @@
+package models
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+)
+
+var db *gorm.DB
+
+func TestMain(m *testing.M) {
+	if _, c := os.LookupEnv(resource.Database); c == false {
```

Comment:
this cause all unit tests in models to be skipped. See https://github.com/almighty/almighty-core/issues/255